AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -411,22 +411,22 @@ - (void)setAsyncDelegate:(id<ASCollectionViewDelegate>)asyncDelegate
 
 - (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeType:(ASLayoutRangeType)rangeType
 {
-  [_collectionNode setTuningParameters:tuningParameters forRangeType:rangeType];
+  [_rangeController setTuningParameters:tuningParameters forRangeMode:ASLayoutRangeModeFull rangeType:rangeType];
```

Comment:
To align with `ASTableView` we should either change this in `ASCollectionView` to `[_layoutController setTuningParameters:...]` as in `ASTableView` or change `ASTableView` to use the range controller via `[_rangeController setTuningParameters:...]`

cc @appleguy @Adlai-Holler